src/components/PendingTxsList/index.tsx

```diff
@@ -0,0 +1,191 @@
+import { ReactElement, useEffect, useState } from 'react'
+import { Icon, Text } from '@gnosis.pm/safe-react-components'
+import {
+  getTransactionQueue,
+  Transaction,
+  TransactionListItem,
+  TransactionSummary,
+} from '@gnosis.pm/safe-react-gateway-sdk'
+import { List } from '@material-ui/core'
+import ChevronRightIcon from '@material-ui/icons/ChevronRight'
+import { Link } from 'react-router-dom'
+
+import { isMultisigExecutionInfo, isTransactionSummary } from 'src/logic/safe/store/models/types/gateway.d'
+import { grey400, sm, primary200 } from 'src/theme/variables'
+import { CustomIconText } from 'src/components/CustomIconText'
+import { getTxTo } from 'src/routes/safe/components/Transactions/TxList/utils'
+
+import styled from 'styled-components'
+import Spacer from 'src/components/Spacer'
+import { useSelector } from 'react-redux'
+import { GATEWAY_URL } from 'src/utils/constants'
+import { currentChainId } from 'src/logic/config/store/selectors'
+import { lastViewedSafe } from 'src/logic/currentSession/store/selectors'
+import { checksumAddress } from 'src/utils/checksumAddress'
+import useOwnerSafes from 'src/logic/safe/hooks/useOwnerSafes'
+import { useTransactionType } from 'src/routes/safe/components/Transactions/TxList/hooks/useTransactionType'
+import { useKnownAddress } from 'src/routes/safe/components/Transactions/TxList/hooks/useKnownAddress'
+import { generateSafeRoute, SAFE_ROUTES } from 'src/routes/routes'
+import { getChainById } from 'src/config'
+import { useAssetInfo } from 'src/routes/safe/components/Transactions/TxList/hooks/useAssetInfo'
+import { TxInfo } from 'src/routes/safe/components/Transactions/TxList/TxCollapsed'
+
+const MAX_TXS_DISPLAY = 5
+
+const PendingTxsList = (): ReactElement => {
+  const chainId = useSelector(currentChainId)
+  const defaultSafe = useSelector(lastViewedSafe) || undefined
```

Comment:
Couple of questions:

1. Can you not use the Safe which is 'open'?
2. Why do you need to fallback to `undefined`? In the effect `!selectedSafe` will work fine with the empty string returned from the selector.

Reply:
> Can you not use the Safe which is 'open'?

Currently we have no open Safe. However when we change the Dashboard url to contain the Safe address we can take it from there

> Why do you need to fallback to undefined? In the effect !selectedSafe will work fine with the empty string returned from the selector.

`useSelector(lastViewedSafe)` can return _null_ but I want `selectedSafe` to be `string | undefined` thus the fallback to `undefined`. Let me know if you know a cleaner way

Reply:
Why not just set `selectedSafe` to also accept `null`?

Reply:
This code was removed